resources/public/api/conf/2.0/application.yaml

```diff
@@ -1127,14 +1139,18 @@ paths:
             example: 2021-06-21T09:00:00Z
         - in: query
           name: page
-          description: This is an optional query parameter, default is 1, that allows the selection of a specific page from a multi-page query.
+          description: >
+            This is an optional query parameter, default is 1, that allows the selection of a specific page from a multi-page query.
+            A check is made on the page parameter to ensure it is a positive number.
           style: form
           schema:
             type: integer
             example: 2
         - in: query
           name: count
-          description: This is an optional query parameter, default is 25, that allows the selection of the number of items per page in a multi-page query.
+          description: >
+            This is an optional query parameter, default is 25, that allows the selection of the number of items per page in a multi-page query.
+            A check is made on the count parameter to ensure it is a positive number.

```

Comment:
Suggestion:
This is an optional query parameter (default value is `25`) that allows the selection of the number of items per page in a multi-page query. If used, you must enter a positive number.